src/tests/dbus-tests/udiskstestcase.py

```diff
@@ -391,6 +391,18 @@ def wipe_fs(self, device):
 
         return False
 
+    def try_unmount(self, path):
+        """Handle unmount and retry if busy"""
+        for _ in range(10):
+            ret, out = self.run_command('umount %s' % path)
+            # the mount may either be unmounted already or not exist anymore
+            if ret == 0 or "not mounted" in out or "no mount point specified" in out or "mountpoint not found" in out:
```

Comment:
I don't like this, but I don't see a better solution so let's hope `umount` won't change the error messages in the future.